shell/lz-bridge/testnet/send.sh

```
@@ -0,0 +1,104 @@
+    echo "Error: not inside a git repository." >&2
+    exit 1
+}
+source "$ROOT_DIR/shell/lib/arguments.sh"
+source "$ROOT_DIR/shell/lib/forge.sh"
+
+load_named_args "$@"
+
+# Load .env so the foundry rpc alias can substitute ALCHEMY_API_KEY.
+[ -f "$ROOT_DIR/.env" ] && set -a && source "$ROOT_DIR/.env" && set +a
+
+validate_text "$chain" "No source chain. Use --chain <sepolia|base-sepolia|arbitrum-sepolia>."
```

Shellcheck (0.11.0)

[warning] chain is referenced but not assigned.

(SC2154)